source/tutorials/batch-inference/index.rst

```diff
@@ -0,0 +1,161 @@
+.. meta::
+    :description: How to do Batch Inference
+
+Batch Inference Tutorial
+========================
+
+In this tutorial you will learn how to create and run a Batch Inference execution in Valohai. This execution will use TensorFlow 2.4.1 to run inference on ten unique images using a pre-trained MNIST model.
+
+.. admonition:: Prerequirements
+    :class: attention
+
+    For this tutorial you will need:
+
+    * Python 3
+
+    We're also going to need two files:
+
+    * a trained MNIST model
+    * some images to classify, packaged in a single zip file
+
+    To make it easy for you they are available here, no need to download them:
+
+    * Model: `<s3://valohai-public-files/tensorflow-batch-inference-tutorial/model.h5>`_
+    * Images: `<s3://valohai-public-files/tensorflow-batch-inference-tutorial/images.zip>`_
+
+    If you want to, you can train the required model by following the tutorial `here <https://www.tensorflow.org/tutorials/quickstart/beginner>`_.
```

Comment:
If you want to, you can train the required model by following our `Valohai 101 tutorial </tutorials/quickstart/>`_.